TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs

```diff
@@ -19,5 +20,10 @@ protected override IMvxApplication CreateApp()
             Mvx.LazyConstructAndRegisterSingleton<IImageHolder, ImageHolder>();
             return new CoreApp();
         }
+
```

Comment:
Why is this change part of this PR and why does it throw a NotImplementedException?

Reply:
I have tried to build all TestProjects and it wasn't implemented so the build failed. I just implemented it then with NotImplementedException, so build will pass, but if someone tries to run this PageRenderer project it will probably crash.

Reply:
Hmmm, it's not causing a build failure (the test projects must be excluded from the CI build) so I think best to revert for the moment 
@Cheesebaron what do you think - leave this in this PR or shift it to provide an actual implementation for UWP?